components/eamxx/docs/developer/source_tree.md

```diff
@@ -1,48 +1,48 @@
-# SCREAM's Source Tree
+# EAMxx's Source Tree
 
-All SCREAM-specific code can be found in `components/eamxx` within the
-[SCREAM repo](https://github.com/E3SM-Project/scream). Here's how things are
+All EAMxx-specific code can be found in `components/eamxx` within the
+[EAMxx repo](https://github.com/E3SM-Project/scream). Here's how things are
 organized:
 
-+ `cime_config`: Tools and XML files for integrating SCREAM with E3SM via the
++ `cime_config`: Tools and XML files for integrating EAMxx with E3SM via the
   CIME framework.
 + `cmake`: CMake functions and macros used by the configuration/build system.
 + `data`: Data files used by our tests.
-+ `docs`: Documentation for the SCREAM project, including design documents,
-  instructions for building and testing SCREAM, and this document.
++ `docs`: Documentation for the EAMxx project, including design documents,
+  instructions for building and testing EAMxx, and this document.
 + `extern`: Source for certain lightweight third-party libraries, embedded
   directly into the repo (and not imported as submodules).
 + `scripts`: Miscellaneous scripts that implement workflows for running tests
   and analyzing performance.
-+ `src`: All C++ source code (and any bridges to Fortran) for SCREAM are stored
++ `src`: All C++ source code (and any bridges to Fortran) for EAMxx are stored
   here. We describe the contents of this directory in greater detail below.
-+ `tests`: Implements standalone, end-to-end tests for various SCREAM
++ `tests`: Implements standalone, end-to-end tests for various EAMxx
   components (RRTMG, HOMME, P3, SHOC, etc).
 
 In addition, you'll notice the following files in `components/eamxx`:
 
-+ `CMakeLists.txt`: The CMake file that defines SCREAM's configuration/build
++ `CMakeLists.txt`: The CMake file that defines EAMxx's configuration/build
   system.
 + `CTestConfig.cmake`: This CTest file contains parameters that determine how
   our test results are reported to the [E3SM CDash Site](http://my.cdash.org/submit.php?project=E3SM).
-+ `README.md`: SCREAM's top-level README file, which describes the project and
++ `README.md`: EAMxx's top-level README file, which describes the project and
```

Comment:
We should now add `mkdocs.yml` as well to the list of additional files in the eamxx folder.